applications/app/controllers/InteractiveController.scala

```diff
@@ -68,121 +64,90 @@ class InteractiveController(
     capiLookup.lookup(path, range = Some(ArticleBlocks))
   }
 
-  def itemResponseToModel(item: Future[ItemResponse])(implicit
+  def toModel(response: ItemResponse)(implicit
       request: RequestHeader,
-  ): Future[Either[(InteractivePage, Blocks), Result]] = {
-    val result = item map { response =>
-      val interactive = response.content map { Interactive.make }
-      val blocks = response.content.flatMap(_.blocks).getOrElse(Blocks())
-      val page = interactive.map(i => InteractivePage(i, StoryPackages(i.metadata.id, response)))
-
-      ModelOrResult(page, response) match {
-        case Left(page)       => Left((page, blocks))
-        case Right(exception) => Right(exception)
-      }
+  ): Either[(InteractivePage, Blocks), Result] = {
+    val interactive = response.content map { Interactive.make }
+    val blocks = response.content.flatMap(_.blocks).getOrElse(Blocks())
+    val page = interactive.map(i => InteractivePage(i, StoryPackages(i.metadata.id, response)))
+
+    ModelOrResult(page, response) match {
+      case Left(page)       => Left((page, blocks))
+      case Right(exception) => Right(exception)
     }
-    result recover convertApiExceptions
-  }
-
-  private def render(model: InteractivePage)(implicit request: RequestHeader) = {
-    val htmlResponse = () => InteractiveHtmlPage.html(model)
-    val jsonResponse = () => views.html.fragments.interactiveBody(model)
-    renderFormat(htmlResponse, jsonResponse, model, Switches.all)
   }
 
   override def canRender(i: ItemResponse): Boolean = i.content.exists(_.isInteractive)
 
-  def renderItemLegacy(item: ItemResponse)(implicit request: RequestHeader): Future[Result] = {
-    itemResponseToModel(Future.successful(item)) map {
-      case Left((model, _)) => render(model)
-      case Right(other)     => RenderOtherStatus(other)
-    }
+  def renderNonDCR(page: InteractivePage)(implicit request: RequestHeader): Future[Result] = {
+    val htmlResponse = () => InteractiveHtmlPage.html(page)
+    val jsonResponse = () => views.html.fragments.interactiveBody(page)
+    val res = renderFormat(htmlResponse, jsonResponse, page, Switches.all)
+    Future.successful(res)
   }
 
-  def renderDCR(item: ItemResponse)(implicit request: RequestHeader): Future[Result] = {
```

Comment:
I dropped all of the `*DCR` suffixes for these methods as DCR is now default and I wanted to emphasise that. This specific method also got renamed to `renderHtml` to make it consistent with `renderJson` and `renderAmp`.